Src/StackifyLib/Utils/HelperFunctions.cs

```diff
@@ -12,8 +12,17 @@ namespace StackifyLib.Utils
     public class HelperFunctions
     {
         static List<string> _BadTypes = new List<string>() { "log4net.Util.SystemStringFormat", "System.Object[]" };
-        static JsonSerializer serializer = new JsonSerializer { ReferenceLoopHandling = ReferenceLoopHandling.Ignore };
-        static JsonSerializerSettings serializerSettings = new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore, ReferenceLoopHandling = ReferenceLoopHandling.Ignore };
+        static JsonSerializerSettings serializerSettings = new JsonSerializerSettings()
+        {
+            NullValueHandling = NullValueHandling.Ignore,
+            ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
+            Converters = new List<JsonConverter>() {
+                    new ToStringConverter("Module", typeof(Module)),
+                    new ToStringConverter("Method", typeof(MethodBase)),
```

Comment:
Instead of MethodBase, then it should be MethodInfo. Then it would also include bad objects like `System.Type`.